pipeline/deploy/cloudformation/morphology_pipeline.yml

```diff
@@ -415,6 +438,49 @@ Resources:
                   }
                 },
                 "ResultPath": "$.landing",
+                "Next": "PlotOriginal"
+              },
+              "PlotOriginal": {
```

Comment:
this is happening outside of landing in order to separate concerns?

Reply:
yep

Reply:
May be I am missing something, but I would not make plotting into separate tasks. Separate functions Yes (for separation of concerns), but not tasks. I am worried, that this would actually complicate pipeline maintenance because of task expansion. If you do decide to use separate tasks, then perhaps we could benefit from some naming convention: naming plotting task according to the preceding task. So instead of the PlotOriginalTask (PlotMorphologyLandingTask) and the same for the python commands.